src/components/OTP.vue

```diff
@@ -378,10 +378,7 @@ export default {
 
     /** Function that verifies the OTP */
     async verifyOTP() {
-      const response = await OTPAuth.verifyOTP(
-        parseInt(this.phoneNumber),
-        parseInt(this.OTPCode)
-      );
+      const response = await OTPAuth.verifyOTP(parseInt(this.phoneNumber), this.OTPCode);
```

Comment:
Why was this change needed?

Reply:
So by converting to int, the code was being sent to the service as a number. In that case, leading zeros were being trimmed and led to incorrect matching. Now, I'm just sending in as a string.